docs/REST API/rest-api.md

```diff
@@ -4118,7 +4118,11 @@ Resends a configuration profile for the specified host.
         "name": "GoogleChrome.pkg",
         "platform": "darwin",
         "version": "125.12.0.3",
-        "self_service": true,
+        "self_service": {
```

Comment:
Dev note: BC break. Self-service is still truthy/false-y with the same semantics as before, but is a nullable object (null means self-service is off) rather than a boolean. Label scoping is now here rather than at the top level.